e2e/tests/LoggedIn/publish.e2e.spec.ts

```diff
@@ -815,6 +816,7 @@ const checkPublicationOnAccountPage = async (
         case 'pending your approval':
             await expect(publicationContainer).toContainText('(Author)');
             await expect(publicationContainer).toContainText('Status: Pending your approval');
+            await expect(publicationContainer.locator(PageModel.myAccount.viewDraftButton)).toBeVisible();
```

Comment:
Is it worth having a test to confirm this doesn't show when in other states? (I think just the approved state).

Reply:
Actually we do want to show it in approved state so that the author can go to the draft and publish it as the next step, but have pushed a commit to be more specific about when the view draft button should and shouldn't show.